frontend/src/routes/manage/Series/Shared.tsx

```diff
@@ -31,7 +31,7 @@ export const makeManageSeriesRoute = (
     render: (series: Series, data: QueryResponse) => JSX.Element,
 ): Route & { url: (args: { seriesId: string }) => string } => (
     makeRoute({
-        url: ({ seriesId }: { seriesId: string }) => `/~manage/series/${keyOfId(seriesId)}/${path}`,
+        url: ({ seriesId }: { seriesId: string }) => `/~manage/series/${keyOfId(seriesId)}${path}`,
```

Comment:
Can you try changing the type of the argument `path` (4 lines up) from `string` to ``` `/${string}` ```? That makes Typescript check that there is always a leading slash passed in the path. Should just work?